### PR TITLE
DSTU 2 Patient updates

### PR DESCRIPTION
**Pull request by amcginn**
Description
----
Changes:
* "id" -> "ID" throughout
  * image
  * image
* reference links updated to https
* search param table
  * image
  * image
* search notes
  * image
  * image

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [x] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/dstu2/individuals/patient.md [hunk 1]
```diff
@@ -226,12 +217,6 @@ Content-Type: application/json
 
 The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
 
-<%= disclaimer %>
-
-### Errors
-
-The common [errors] and [OperationOutcomes] may be returned.
-
 [Time of day of birth]: http://hl7.org/fhir/DSTU2/extension-patient-birthtime.html
```
Comment (RicardoCrespo-ora) [inline content/millennium/dstu2/individuals/patient.md:220]: Might want to change these http for https

**Review by RicardoCrespo-ora — APPROVED** [review]

File: content/millennium/dstu2/individuals/patient.md [hunk 2]
```diff
@@ -9,24 +9,30 @@ title: Patient | DSTU 2 API
...
 * [Date of Birth](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.birthDate){:target="_blank"}
 * [Deceased](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.deceased_x_){:target="_blank"}
 * [Address](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.address){:target="_blank"}
-* [Communication (preferred language)](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.communication.language){:target="_blank"}
+* [Communication language](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.communication.language){:target="_blank"}
 * [Marital status](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.maritalStatus){:target="_blank"}
-* [Primary Care Providers](http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.careProvider){:target="_blank"}
-* [Multiple Birth Information](http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.multipleBirth_x_){:target="_blank"}
+* [Care Provider](http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.careProvider){:target="_blank"}
+* [Multiple Birth](http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.multipleBirth_x_){:target="_blank"}
```
Comment (kckeim) [inline content/millennium/dstu2/individuals/patient.md:29]: You'll want to double-check for https-ing here.
You got a couple above this list, but starting at `Identifier` they're http

File: content/millennium/dstu2/individuals/patient.md (hunk 1 above)
Comment (kckeim) [reply content/millennium/dstu2/individuals/patient.md:220]: Agreed

File: lib/resources/dstu2/patient.yaml [hunk 3]
```diff
@@ -218,15 +253,15 @@ fields:
       "maritalStatus": {
         "coding": [
           {
-            "system": "http://hl7.org/fhir/v3/NullFlavor",
-            "code": "UNK",
-            "display": "Unknown"
+            "system": "http://terminology.hl7.org/CodeSystem/v3-MaritalStatus",
+            "code": "S",
+            "display": "Never Married"
```
Comment (kckeim) [inline lib/resources/dstu2/patient.yaml:258]: Good change to have a working example here. :)

File: lib/resources/dstu2/patient.yaml [hunk 4]
```diff
@@ -272,8 +307,8 @@ fields:
         }
       ]
     }
-  note: careProvider must be a reference to a Practitioner
-  url: http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.careProvider
+  note: The careProvider must be a reference to a Practitioner
```
Comment (kckeim) [inline lib/resources/dstu2/patient.yaml:310]:
```suggestion
  note: The careProvider must be a reference to a Practitioner.
```

**Review by kckeim — APPROVED** [review]
Wanted to nit about some of the Notes under the Parameter table, but I don't think it's necessary stuff.

Just a few other notes provided. 
Nothing game-breaking.

**Review by cldeitrich — APPROVED** [review]
[OHAIUAD-205303](https://jira3.cerner.com/browse/OHAIUAD-205303): Approved for style.

File: content/millennium/dstu2/individuals/patient.md (hunk 2 above)
Comment (amcginn) [reply content/millennium/dstu2/individuals/patient.md:29]: apparently completely missed checking this. fixing some bad links too.

File: lib/resources/dstu2/patient.yaml (hunk 3 above)
Comment (amcginn) [reply lib/resources/dstu2/patient.yaml:258]: I saw this a while back and like, whaaat?? 🙄

**Comment by SukeshKonjeti** [comment]
after changes published
Screenshot 2024-02-02 at 5 22 28 PM

Screenshot 2024-02-02 at 5 23 19 PM

Screenshot 2024-02-02 at 5 23 35 PM

**Pull request merged**